core/src/main/java/com/novoda/noplayer/player/PlayerFactory.java

```diff
@@ -41,21 +43,25 @@ public Player create() {
     }
 
     public Player create(DrmType drmType, DrmHandler drmHandler) {
+        return create(drmType, drmHandler, USE_SECURE_CODEC);
+    }
+
+    public Player create(DrmType drmType, DrmHandler drmHandler, boolean useSecureCodec) {
         for (PlayerType player : prioritizedPlayerTypes) {
             if (player.supports(drmType)) {
-                return createPlayerForType(player, drmType, drmHandler);
+                return createPlayerForType(player, drmType, drmHandler, useSecureCodec);
             }
         }
         throw UnableToCreatePlayerException.unhandledDrmType(drmType);
     }
 
-    private Player createPlayerForType(PlayerType playerType, DrmType drmType, DrmHandler drmHandler) {
+    private Player createPlayerForType(PlayerType playerType, DrmType drmType, DrmHandler drmHandler, boolean useSecureCodec) {
```

Comment:
I think that its ok to leave internal implementation with a boolean (as is a less big deal) but with the public API creating two different methods is clearer. If its not widely used its even better, you keep the normal method standardised and another with that is fit to purpose with a good naming ;)